buddyboss/buddyboss-platform
# Removed hidden profile type users from search result count

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1322 .

### How to test the changes in this Pull Request:

1. Create 2 Profile Types: one Hide all members of this type from Members Directory is enabled and the other one is not
2. Assign Profile Types to 2 different users.
3. In the Profile Fields of the two users, enter a value to be searched
4. On frontend, search that value
5. See error on the results

### Proof Screenshots or Video

https://www.loom.com/share/67ca4ab5edb2447aab64b3bc9c9100e1

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
